app/helpers/pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PagesHelper
```

Comment:
Let's all start getting in the habit of deleting any helper files that we're not using.